Makefile

```diff
@@ -37,8 +37,19 @@ technical-docs:  ## build the technical docs
 guides:	swagger ## build the developer guide docs
 	cd docs/guides; make clean html
 
+# (IS_OPENEDX_TRANSLATIONS_WORKFLOW) is set to "yes" when openedx-translations repository extracts translations.
```

Comment:
```suggestion
# (IS_OPENEDX_TRANSLATIONS_WORKFLOW) is set to "yes" in the `extract-translation-source-files` GitHub actions workflow on the `openedx-translations` repository
```

might even be worth it to throw a link to the repo/workflow file in here

Reply:
done